src/collections/_documentation/platforms/javascript/react.md

```diff
@@ -50,10 +50,10 @@ class ExampleBoundary extends Component {
     }
 
     componentDidCatch(error, errorInfo) {
```

Comment:
class is es6, but babel or typescript is required to use React, so using es6 for react code is fine.